octokit/users.py

```diff
@@ -0,0 +1,14 @@
+import uritemplate
+
+class Users(object):
+    def get_user(self, inputUser=None, **kwargs):
+        if inputUser:
+            return self.user(user=inputUser)
+        else:
+            return self.current_user
+
+    def get_all_users(self):
+        url = uritemplate.expand(self.user.url, {})
+        if(url[-1] == '/'):
+            url = url[:-1]
+        return self.paginate(url[:-1], {})
```

Comment:
So with our current scheme, our mixin classes shouldn't need to handle pagination at all. The idea was to intercept responses in our Pagination mixin class and auto-paginate those so our convenience classes don't need to worry about pagination.